### ISSUE TITLE
the lib super sensetive to GHC version

### ISSUE THREAD

**Issue by yaitskov** [haskell-github-trust/true-name#3]
I tried 8.10.1, 8.10.4, 8.8.4 and 8.4.4
Only 8.4.4 is working. Others fail on step of building true-name.

```
Warning: /home/dan/.stack/config.yaml: Unrecognized field in ConfigMonoid: compiler
WARNING: Ignoring p's bounds on template-haskell (==2.14.0.0); using template-haskell-2.16.0.0.
```

last applicable template Haskell library version is not compatible with base for ghc 8.10

**Comment by yaitskov** [comment]
I wanted to use this library to debug GHC api (parseDynamicFlags returns type without Outputable/Show instances).

**Comment by nuttycom** [comment]
Fixed in release 0.2.0.0
